searchlib/src/vespa/searchlib/engine/request.cpp

```diff
@@ -5,8 +5,8 @@
 
 namespace search::engine {
 
-Request::Request(const fastos::TimeStamp &start_time)
-    : _startTime(start_time),
+Request::Request(RelativeTime && relativeTime)
```

Comment:
should probably pass by value here as well

Reply:
Fixed